src/main/java/org/mockito/internal/matchers/text/ValuePrinter.java

```diff
@@ -41,7 +41,9 @@ public static String print(Object value) {
      * @return neatly formatted value list
      */
     public static String printValues(String start, String separator, String end, Iterator values) {
-        //TODO SF null check
+        if(start == null || separator == null || end == null || values == null){
+            return null;
```

Comment:
Returning null seems dangerous for future usages. Maybe it is better to return empty string or use an exception to notigy the developer he made an error.